guide/controller-plugin.md

```diff
@@ -31,13 +31,13 @@ After its loading, some controller plugin functions (entry points) are called by
 A set of entry points have to be defined to let the controller plugin work smoothly.
 Some of these entry points are required and some are optional.
 
-The `Robot` node defines the location of the controller plugin through its *window* and its *remoteControl* fields (cf. reference manual).
+The [Robot](../reference/robot.md) node defines the location of the controller plugin through its *window* and its *remoteControl* fields (cf. [Reference Manual](../reference/index.md)).
```

Comment:
These links to the reference manual are a little bit useless. I would either remove them or link to the robot node of the reference manual.

Reply:
Yes I think we can remove it as 'Robot' already points to the reference manual.